k8s/aat/common/idam/idam-api.yaml

```
@@ -15,13 +15,20 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: idam-api
-    version: 0.2.2
+    version: 0.2.3
   values:
     java:
-      image: hmctspublic.azurecr.io/idam/api:aat-16d37681
-      ingressHost: idam-api.service.core-compute-aat.internal
+      image: hmctspublic.azurecr.io/idam/api:prod
```

is prod tag a thing? normally we just put latest in for the first merge, and then flux will automatically update it
```suggestion
      image: hmctspublic.azurecr.io/idam/api:latest
```